src/chains/ethereum/ethereum/src/data-managers/account-manager.ts

```diff
@@ -9,86 +9,96 @@ import { Quantity, Data } from "@ganache/utils";
 import { Address } from "@ganache/ethereum-address";
 import { decode } from "@ganache/rlp";
 import Blockchain from "../blockchain";
+import { PendingBlock } from "@ganache/ethereum-block";
 export default class AccountManager {
   #blockchain: Blockchain;
 
   constructor(blockchain: Blockchain) {
     this.#blockchain = blockchain;
   }
 
-  public async get(
-    address: Address,
-    blockNumber: Buffer | Tag = Tag.latest
-  ): Promise<Account | null> {
-    const raw = await this.getRaw(address, blockNumber);
-    if (raw == null) return null;
-    return Account.fromBuffer(raw);
+  /**
+   * Creates a copy of the trie that was used to mine the specified
+   * `blockNumber`. Sets the context of the trie to the state root and number
+   * of `blockNumber`.
+   * @param blockNumber
+   * @returns
```

Comment:
```suggestion
   * @returns { trie: GanacheTrie }
```



Additionally, every time we call this, we immediately destructure the return object. Can we just return `GanacheTrie`?

Reply:
Ahh good find! I think originally this returned multiple values so I threw it in an object. Fixed!